src/backend/el.d

```diff
@@ -0,0 +1,221 @@
+/**
+ * Compiler implementation of the
+ * $(LINK2 http://www.dlang.org, D programming language).
+ *
+ * Copyright:   Copyright (C) 1985-1995 by Symantec
+ *              Copyright (c) 1999-2016 by Digital Mars, All Rights Reserved
+ * Authors:     $(LINK2 http://www.digitalmars.com, Walter Bright)
+ * License:     backendlicense.txt
+ * Source:      $(DMDSRC backend/_el.d)
+ */
+
+module ddmd.backend.el;
+
+import ddmd.backend.cdef;
+import ddmd.backend.cc;
+import ddmd.backend.type;
+
+import ddmd.backend.cc : Symbol;
```

Comment:
Why don't you stop adding double definitions then?  I do remember having problems with forward references when I was first translating the compiler, but AFAIK they were all resolved years ago.  If you avoid introducing the conflicting symbols then it should be as simple as importing every required module here.